test/tools/evmoneRunner.h

```diff
@@ -0,0 +1,48 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <deps/evmone/include/evmone/evmone.h>
+#include <string>
+
+namespace external
+{
+namespace evmone
+{
+class EvmOneVM : public evmc_context
```

Comment:
```suggestion
class EvmOneVM: public evmc_context
```

Reply:
Which namespace is `evmc_context` from?

Reply:
It's in the global namespace that is included from a evmone header called `evmc.h`

https://github.com/ethereum/evmc/blob/37a5b8adf650824083c146f229c6426b98a731ea/include/evmc/evmc.h#L654-L667